node/pkg/node/node_test.go

```diff
@@ -1004,6 +1004,34 @@ func BenchmarkCrypto(b *testing.B) {
 		})
 	})
 
+	b.Run("libp2p (RSA)", func(b *testing.B) {
```

Comment:
This is a nice to have, but please include a comment above giving context on why it's here, otherwise it will get confusing in the future to see benchmarks for cryptography algorithms that are not used anywhere in the codebase.